aquadoggo/src/graphql/client/mutation.rs

```diff
@@ -29,66 +30,111 @@ impl Mutation {
         )]
         operation_encoded_param: String,
     ) -> Result<PublishEntryResponse> {
+        let store = ctx.data::<SqlStorage>()?;
+        let tx = ctx.data::<ServiceSender>()?;
+
         // Parse and validate parameters
         let args = PublishEntryRequest {
             entry_encoded: EntrySigned::new(&entry_encoded_param)?,
             operation_encoded: OperationEncoded::new(&operation_encoded_param)?,
         };
 
-        let store = ctx.data::<SqlStorage>()?;
-        store.publish_entry(&args).await.map_err(Error::from)
+        // Validate and store entry in database
+        // @TODO: Check all validation steps here for both entries and operations. Also, there is
```

Comment:
Yeh, there's an issue for this: https://github.com/p2panda/p2panda/issues/301

Reply:
Thats cool, though, that issue does not seem to cover validation of data but rather talks about when to throw / not throw? (I somehow have the feeling there are loads of validation cases we didn't cover yet, but maybe I'm mistaken, would be nice to have a table someday where we validate what and then just go through the code to see if its implemented [similar to how OpenMLS does it, they even have identifiers for each validation, related issue: https://github.com/p2panda/handbook/issues/126]).